Kismet stuck in upgrade failure

Fresh ptf install, all modules updated. Ubuntu 20.04 (x64)

```
ptf> use modules/wireless/kismet
ptf:(modules/wireless/kismet)>run



Module Author:         Dave Kennedy (HackingDave)
Module Description:    This module will install/update kismet, a tool for wireless auditing
-------------------------------------------------------------------------------------
INSTALL_TYPE:           GIT
REPOSITORY_LOCATION:    https://github.com/kismetwireless/kismet
INSTALL_LOCATION:       /pentest/wireless/kismet/
-------------------------------------------------------------------------------------
[*] Detected installation already. Going to upgrade for you.
[*] Updating modules/wireless/kismet , be patient while git pull is initiated
[*] Running after commands for post installation requirements.
[*] Sending after command: cd /pentest/wireless/kismet/
[*] Sending after command: ./configure
checking build system type... x86_64-unknown-linux-gnu
checking host system type... x86_64-unknown-linux-gnu
checking for gcc... gcc
checking whether the C compiler works... yes
checking for C compiler default output file name... a.out
checking for suffix of executables... 
checking whether we are cross compiling... no
checking for suffix of object files... o
checking whether we are using the GNU C compiler... yes
checking whether gcc accepts -g... yes
checking for gcc option to accept ISO C89... none needed
checking for gcc option to accept ISO C99... none needed
checking for gcc option to accept ISO Standard C... (cached) none needed
checking for g++... g++
checking whether we are using the GNU C++ compiler... yes
checking whether g++ accepts -g... yes
checking for a BSD-compatible install... /usr/bin/install -c
checking whether make sets $(MAKE)... yes
checking how to run the C preprocessor... gcc -E
checking for platform-specific compiler flags... none needed
checking gcc version... 9
checking whether g++ supports C++17 features with -std=gnu++17... yes
checking how to run the C++ preprocessor... g++ -std=gnu++17 -E
checking for grep that handles long lines and -e... /usr/bin/grep
checking for egrep... /usr/bin/grep -E
checking for ANSI C header files... yes
checking for sys/types.h... yes
checking for sys/stat.h... yes
checking for stdlib.h... yes
checking for string.h... yes
checking for memory.h... yes
checking for strings.h... yes
checking for inttypes.h... yes
checking for stdint.h... yes
checking for unistd.h... yes
checking whether byte ordering is bigendian... no
checking python3 module: setuptools... yes
checking for dwarf_begin in -ldw... no
checking for bfd_alloc in -lbfd... no
checking unwind.h usability... yes
checking unwind.h presence... yes
checking for unwind.h... yes
checking execinfo.h usability... yes
checking execinfo.h presence... yes
checking for execinfo.h... yes
checking for stdint.h... (cached) yes
checking for strerror_r() return... char *
checking for accept() addrlen type... socklen_t
checking for pipe2... yes
checking for special C compiler options needed for large files... no
checking for _FILE_OFFSET_BITS value needed for large files... no
checking for dlopen in -ldl... yes
checking for deflate in -lz... yes
checking for pthread_mutex_timedlock... no
checking for libm math function in std libs... yes
checking for main in -lstdc++... yes
checking for group 'root'... yes
checking for MHD_start_daemon in -lmicrohttpd... no
configure: error: libmicrohttpd is needed and is not available
[*] Sending after command:  make
make: *** No targets specified and no makefile found.  Stop.
[*] Sending after command: make install
make: *** No rule to make target 'install'.  Stop.
[*] Completed running after commands routine..
[*] Finished Installing! Enjoy the tool installed under: /pentest/wireless/kismet/
[*] Running updatedb to tidy everything up.
ptf:(modules/wireless/kismet)>

```

--- a/modules/wireless/kismet.py
+++ b/modules/wireless/kismet.py
@@ -14,13 +14,13 @@ DESCRIPTION="This module will install/update kismet, a tool for wireless auditin
 INSTALL_TYPE="GIT"
 
 # LOCATION OF THE FILE OR GIT/SVN REPOSITORY
-REPOSITORY_LOCATION="https://github.com/kismetwireless/kismet" 
+REPOSITORY_LOCATION="https://github.com/kismetwireless/kismet"
 
 # WHERE DO YOU WANT TO INSTALL IT
 INSTALL_LOCATION="kismet"
 
 # DEPENDS FOR DEBIAN INSTALLS
-DEBIAN="git,libnl-3-dev,libnl-cli-3-dev"
+DEBIAN="git,libnl-3-dev,libnl-cli-3-dev,libmicrohttpd-dev,libprotobuf-dev,protobuf-compiler,libprotobuf-c-dev,libprotoc-dev,protobuf-c-compiler"
 
 # DEPENDS FOR FEDORA INSTALLS
 FEDORA="git"

